.sphinx/lxd-doc/README.md

```diff
@@ -0,0 +1,177 @@
+# lxddoc
+
+A small CLI to parse comments in a Golang codebase meant to be used for a documentation tool (like Sphinx for example).
+
+It parses the comments from the AST and extracts their documentation.
+
+# Usage
+
+    $ lxd-doc -h
+    Usage of lxd-doc:
+    -e value
+        Path that will be excluded from the process
+
+# Formatting
+
+A comment is formatted this way:
+
+```go
+	// lxddoc:generate(group=cluster, key=scheduler.instance)
+	//
+	//  <Possibly a very long documentation on multiple lines with Markdown tables, etc.>
+	// ---
+	//  shortdesc: Possible values are all, manual and group. See Automatic placement of instances for more information.
+	//  <ANY_KEY>: <ANY_VALUE>
+    //  condition: container
+	//  default: `all`
+	//  type: integer
+	//  liveupdate: `yes`
```

Comment:
```suggestion
	//  shortdesc: Possible values are all, manual and group. See Automatic placement of instances for more information.
        //  condition: container
	//  default: `all`
	//  type: integer
	//  liveupdate: `yes`
	//  <ANY_KEY>: <ANY_VALUE>
```

We should have all of them at the same indentation. ;)
And imo it makes most sense to have the "any key" one at the end.